playdate/sound.lua

```diff
@@ -96,20 +169,26 @@ function fileplayer.meta:stop()
   self.data:stop()
 end
 
-function fileplayer.meta:pause(value)
+function fileplayer.meta:pause()
   self.data:pause()
 end
 
 function fileplayer.meta:isPlaying()
   return self.data:isPlaying()
 end
 
-function fileplayer.meta:setVolume(value)
-  self.data:setVolume(value)
+function fileplayer.meta:setVolume(left, right, fadeSeconds, fadeCallback, arg)
+  @@ASSERT(right == nil, "[ERR] Parameter right is not yet implemented.")
+  @@ASSERT(fadeSeconds == nil, "[ERR] Parameter fadeSeconds is not yet implemented.")
+  @@ASSERT(fadeCallback == nil, "[ERR] Parameter fadeCallback is not yet implemented.")
+  @@ASSERT(arg == nil, "[ERR] Parameter arg is not yet implemented.")
+
+  self.data:setVolume(left)
 end
 
 function fileplayer.meta:getVolume()
-  return self.data:getVolume()
+  -- TODO: should have two return values; left, right
+  return self.data:getVolume(), self.data:getVolume()
```

Comment:
@fletchmakes I can't actually tell from the docs if the PD SDK returns two _different_ values for stereo, or just the same value twice like you have here? 

If it's the former, I'm not clear how those values should be different? 

If its the latter, then this can easily be implemented with:
```suggestion
  if self.data::getChannelCount() == 1 then
      return self.data:getVolume()
  else
      return self.data:getVolume(), self.data:getVolume()
  end
```

Reply:
The SDK specifies that when the file is playing in stereo mode, the first return value is the "left" volume, the second is the "right". My comment was intended to say "eventually we'll need to update this function to properly return different values for left and right, but for now, returning a single volume twice will have to be sufficient".
https://sdk.play.date/2.7.1/Inside%20Playdate.html#m-sound.fileplayer.getVolume